lib/modules/platform/gerrit/utils.spec.ts

```diff
@@ -0,0 +1,309 @@
+    });
+
+    it('hashtag with "sourceBranch-" prefix', () => {
+      const change = partial<GerritChange>({
+        hashtags: ['other', 'sourceBranch-renovate/dependency-1.x', 'another'],
+      });
+      expect(utils.extractSourceBranch(change)).toBe('renovate/dependency-1.x');
+    });
+  });
+
+  describe('findPullRequestBody()', () => {
+    it('find pull-request-body', () => {
+      const change = partial<GerritChange>({
+        messages: [
+          partial<GerritChangeMessageInfo>({
+            id: '9d78ac236714cee8c2d86e95d638358925cf6853',
+            tag: 'pull-request',
+            message: 'Patch Set 1:\n\nOld PR-Body',
+          }),
+          partial<GerritChangeMessageInfo>({
+            id: '1d17c930381e88e177bbc59595c3ec941bd21028',
+            tag: 'pull-request',
+            message: 'Patch Set 12:\n\nLast PR-Body',
+          }),
+          partial<GerritChangeMessageInfo>({
+            id: '9d78ac236714cee8c2d86e95d638358925cf6853',
+            message: 'other message...',
+          }),
+        ],
+      });
+      expect(utils.findPullRequestBody(change)).toBe('Last PR-Body');
+    });
+
+    it('no pull-request-body message found', () => {
+      const change = partial<GerritChange>({});
+      expect(utils.findPullRequestBody(change)).toBeUndefined();
+      change.messages = [];
+      expect(utils.findPullRequestBody(change)).toBeUndefined();
+      change.messages = [
+        partial<GerritChangeMessageInfo>({
+          tag: 'other-tag',
+          message: 'message',
+        }),
+      ];
+      expect(utils.findPullRequestBody(change)).toBeUndefined();
+    });
+  });
+
+  describe('mapBranchStatusToLabel()', () => {
+    const labelWithOne: GerritLabelTypeInfo = {
+      values: { '-1': 'rejected', '0': 'default', '1': 'accepted' },
+      default_value: 0,
+    };
+
+    it.each([
+      ['red' as BranchStatus, -1],
+      ['yellow' as BranchStatus, -1],
+      ['green' as BranchStatus, 1],
+    ])(
+      'Label with +1/-1 map branchState=%p to %p',
+      (branchState, expectedValue) => {
+        expect(mapBranchStatusToLabel(branchState, labelWithOne)).toEqual(
+          expectedValue
+        );
+      }
+    );
+
+    const labelWithTwo: GerritLabelTypeInfo = {
+      values: {
+        '-2': 'rejected',
+        '-1': 'disliked',
+        '0': 'default',
+        '1': 'looksOkay',
+        '2': 'approved',
+      },
+      default_value: 0,
+    };
+
+    it.each([
+      ['red' as BranchStatus, -2],
+      ['yellow' as BranchStatus, -2],
+      ['green' as BranchStatus, 2],
+    ])(
+      'Label with +2/-2, map branchState=%p to %p',
+      (branchState, expectedValue) => {
+        expect(mapBranchStatusToLabel(branchState, labelWithTwo)).toEqual(
+          expectedValue
+        );
+      }
+    );
+  });
+
+  describe('mapBranchStateContextToLabel()', () => {
+    const labelConfig: RepoGlobalConfig['gerritLabelMapping'] = {
+      stabilityDaysLabel: 'Stability',
```

Comment:
`stabilityDays` is now `minimumReleaseAge`. Please update the docs and code. 😉 

https://docs.renovatebot.com/configuration-options/#minimumreleaseage